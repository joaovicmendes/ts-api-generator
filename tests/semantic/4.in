// Deve reportar que identificador de rota deveria/n√£o deveria existir
model [
    Student {
        name: string
        email: string
        ra: number
    }
    ExampleClass {
        student: Student
        grade: number
    }
]
endpoints [
    Student {
        GET '/students'
        GET '/students/:ra'
        POST '/students:ra'
        PUT '/students/'
        DELETE '/students/'
    }
    ExampleClass {
        GET 'class/:grade'
    }
]
