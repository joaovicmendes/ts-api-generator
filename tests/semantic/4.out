Linha 17: método POST não deveria conter um identificador
Linha 18: método PUT deveria conter um identificador
Linha 19: método DELETE deveria conter um identificador
Compilação interrompida.
